dart/looker_sdk/example/main.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:io';
+import 'dart:typed_data';
+import 'package:looker_sdk/index.dart';
+import 'package:dotenv/dotenv.dart' show load, env;
+
+void main() async {
+  load();
+  var sdk = await createSdk();
+  await runLooks(sdk);
+  await runDashboardApis(sdk);
+  await runConnectionApis(sdk);
+}
+
+Future<LookerSDK> createSdk() async {
+  return await Sdk.createSdk({
+    'base_url': env['URL'],
+    'verify_ssl': false,
+    'credentials_callback': credentialsCallback
+  });
+}
+
+Future<void> runLooks(LookerSDK sdk) async {
+  try {
+    var looks = await sdk.ok(sdk.allLooks());
+    if (looks.isNotEmpty) {
+      for (var look in looks) {
+        print(look.title);
+      }
+      var look = await sdk.ok(sdk.runLook(looks[looks.length - 1].id, 'png'));
```

Comment:
FWIW if you want to have a test for binary payloads, `content_thumbnail` is pretty quick for PNG